src/one_dragon/base/operation/application/application_run_context.py

```
@@ -422,8 +423,8 @@ def run_application(
             self.current_application = app
 
             op_result = app.execute()
-        except Exception:
-            log.error("运行应用 {} 失败", app_id)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)